65.md

```diff
@@ -4,73 +4,56 @@ NIP-65
 Relay List Metadata
 -------------------
 
-`draft` `optional` `author:mikedilger`
+`draft` `optional` `author:mikedilger` `author:vitorpamplona`
 
-A special replaceable event meaning "Relay List Metadata" is defined as an event with kind `10002` having a list of `r` tags, one for each relay the author uses to either read or write to.
+Defines a replaceable event using `kind:10002` to advertise preferred relays for discovering a user's content and receiving fresh content from others.
```

Comment:
Those two reasons sound the same to me.  How about "for finding a user's content and for ensuring content that references them can be found by them."

Reply:
Not only by them, right? By anybody. 

The idea I was trying to convey in the latter part, is that this NIP tells other users where to broadcast events that involve the author, which is a novelty of this NIP. We don't have that logic anywhere else in Nostr. It's not about them finding it. It's about how other users should behave if they implement this NIP. The finding is a consequence of that behavior.